Source/Neptune.Web/Views/RegionalSubbasin/RegionalSubbasinGridSpec.cs

```diff
@@ -36,11 +36,11 @@ public class RegionalSubbasinGridSpec : GridSpec<Models.RegionalSubbasin>
         public RegionalSubbasinGridSpec()
         {
             Add(string.Empty, x => UrlTemplate.MakeHrefString(x.GetDetailUrl(), "View", new Dictionary<string, string> { { "class", "gridButton" } }), 50, DhtmlxGridColumnFilterType.None);
-
+            Add("RSB ID", x => x.RegionalSubbasinID.ToString(), 100, DhtmlxGridColumnFilterType.Numeric);

```

Comment:
take the ToString() out of the RSB ID and Catchment ID in OC Survey columns and it will automatically resolve as a numeric and thus wouldn't need the DhtmlxGridColumnFilterType.Numeric)